APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,212 @@
+An Updated Model for the Affiliated Package Ecosystem
+-----------------------------------------------------
+
+author: Erik Tollerud, 2017 Astropy Coordination Meeting Attendees
+
+date-created: 2018 February 22
+
+date-last-revised: 2018 February 22
+
+type: Process
+
+status: Discussion
+
+
+Abstract
+--------
+
+A revision of the concept of "affiliated packages" is proposed.  The idea of
+affiliated packages has continued mostly unaltered since the original Vision
+for Astropy.  However, time has revealed some potential improvements in how the
+ecosystem is structured and the review process for affiliated packages.  Hence,
+this APE proposes to make three changes:
+
+* This APE creates the concept of "coordinated" packages - affiliated-like packages
+  that are more closely overseen by the Astropy Project. While apparently
+  significant, this change is mainly a recognition of de facto status that some
+  affiliated packages already have.
+* The APE changes the review process to a more explicit criteria-based review
+  process - again, this is primarily simply formalizing a process that was
+  already in place in the coordination committee.
+* It lays out guidelines for the eventual goal of converting the
+  affiliated package review process into a more "peer-review" style, where
+  other members of the community can contribute to the review.
+
+
+Detailed description
+--------------------
+
+Motivation
+==========
+
+From its inception, Astropy has included an ecosystem of astronomy packages
+beyond the ``astropy`` core package (see the
+`original vision <http://docs.astropy.org/en/stable/development/vision.html>`_).
+The process by which these "affiliated" packages are reviewed and accepted has
+already changed a few times since the project's inception as the ecosystem has
+grown. However, the current state has several significant limitations.
+
+First, the affiliated packages include a mix of two different kinds of packages.
+Some packages have been developed as a fairly integrated effort alongside the
+core package, meaning both feature planning assuming the existence of the
+affiliated packages, as well as significant overlap in the core development
```

Comment:
but this would remove the fact that these packages are overseen by the same coordination committee rather than being the product of one single developer. And when needed new people can be put in charge of maintaining these packages while that's not really the case for a normal affiliated package.

Reply:
Not really: according to the text later on, those people would become part of the astropy team. I just thought one should not suggest that one already has to be part of the astropy core team to have a core affiliated package (even though in practice this may usually be the case; it is really just a matter of feeling that the current text gives the impression of an exclusive club).

Reply:
ah, ok I agree that this shouldn't suggest being a closed "club" either. But difficult to see that one can "have a core affiliated package", as the nature of those package is that they come from the community need rather than applied for affiliated status by someone "owning" it.

Reply:
Yep, I see this point - I took @mhvk's wording and massaged it a bit.  Since me pushing up the new draft will zap this comment chain, maybe do another review if you have more suggestions on the new wording, @mhvk or @bsipocz ?